sherpa/optmethods/optfcts.py

```diff
@@ -583,6 +735,204 @@ def random_start(xmin, xmax):
 def neldermead(fcn, x0, xmin, xmax, ftol=EPSILON, maxfev=None,
                initsimplex=0, finalsimplex=9, step=None, iquad=1,
                verbose=0):
+    r"""Nelder-Mead Simplex optimization method.
+
+    The Nelder-Mead Simplex algorithm, devised by J.A. Nelder and
+    R. Mead [1]_, is a direct search method of optimization for
+    finding local minimum of an objective function of several
+    variables. The implementation of Nelder-Mead Simplex algorithm is
+    a variation of the algorithm outlined in [2]_ and [3]_. As noted,
+    terminating the simplex is not a simple task:
+
+    "For any non-derivative method, the issue of termination is
+    problematical as well as highly sensitive to problem scaling.
+    Since gradient information is unavailable, it is provably
```

Comment:
```suggestion
    Since gradient information is unavailable, it is probably
```
(Unless the typo is part of the original reference)

Reply:
I have no idea. This stuff is just copied over from `__init__.py`

Reply:
I'm pretty sure the text is correct: https://www.merriam-webster.com/dictionary/provably

Reply:
It always pays-off to learn from a native speaker. I learnt a new word today.